Tests iOS/SignOutTests.swift

```diff
@@ -44,4 +44,35 @@ class SignOutTests: XCTestCase {
         try server.stop()
         app.terminate()
     }
+    
+    func test_tappingSignOutshowsLogin() {
+        app.tabBar.settingsButton.wait().tap()
+        tap_SignOut()
+        
+        let account = XCUIApplication()
+        account.alerts["Are you sure?"].scrollViews.otherElements.buttons["Sign Out"].wait().tap()
+        app.accountView.loginText.exists
+    }
+    
+    func test_tappingSignOutCancelshowsAccountsMenu() {
+        app.tabBar.settingsButton.wait().tap()
+        tap_SignOut()
+        
+        let account = XCUIApplication()
+        account.alerts["Are you sure?"].scrollViews.otherElements.buttons["Cancel"].wait().tap()
+        let cellCount = account.cells.count
+        XCTAssertTrue(cellCount > 0)
```

Comment:
we can also use `XCTAssertGreaterThan(cellCount, 0)` but either works!

Reply:
changed it, I like that suggestion thanks